### PR TITLE
Drive to position

### LINKED ISSUE #5: Create distance-based auto drive command
To test out Swerve drive and to help learn how we can start making commands that we can use to make an effective Autonomous mode, we can create a new command to pick a direction and a distance for the robot to travel in that direction.

### PR DESCRIPTION
**Pull request by tri-vu-12**
Added DriveToPosition command

Fixes #5

### CODE REVIEW

File: src/main/java/frc/robot/commands/DriveToPosition.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    var currentPose = m_drive.getPose();
+    var difference = m_targetPose.getTranslation().minus(currentPose.getTranslation());
+    return difference;
+  }
+
+  private Rotation2d getRotationFromTarget() {
+    var currentPose = m_drive.getPose();
+    var currentRotation = currentPose.getRotation();
+    var targetRotation = m_targetPose.getRotation();
+    SmartDashboard.putString("targetRotation", targetRotation.toString());
```
Comment (jnallard) [inline src/main/java/frc/robot/commands/DriveToPosition.java]: Can you remove the debug smartdashboard lines?

File: src/main/java/frc/robot/commands/DriveToPosition.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    var difference = getDistanceFromTarget();
+    var rotationDifference = getRotationFromTarget();
+    var isXGood = Math.abs(difference.getX()) < 0.01;
+    var isYGood = Math.abs(difference.getY()) < 0.01;
+    var isRotationGood = Math.abs(rotationDifference.getDegrees()) < 0.1;
+    return isXGood && isYGood && isRotationGood;
```
Comment (jnallard) [inline src/main/java/frc/robot/commands/DriveToPosition.java:88]: I like this!

**Review by jnallard — CHANGES_REQUESTED** [review]
Mostly good, but let's remove the smartdashboard statements

File: src/main/java/frc/robot/commands/DriveToPosition.java (hunk 1 above)
Comment (tri-vu-12) [reply src/main/java/frc/robot/commands/DriveToPosition.java]: Yes

**Review by jnallard — APPROVED** [review]
Good job! And thanks for making the changes

**Pull request merged**